en/Ciphers/rsa_cipher.md

```diff
@@ -0,0 +1,162 @@
+# RSA Cryptography Algorithm
+
+## About RSA Algorithm
+RSA (Rivest–Shamir–Adleman) is one of the first public-key cryptosystems and is widely used for secure data transmission. In such a cryptosystem, the encryption key is public and distinct from the decryption key which is kept secret (private).
+
+## Algorithm
+
+1. Select 2 Prime Numbers - **p & q**
+2. Calculate `n` as $$n = p * q$$
+3. Calculate Euler's Totient Function of n, $$φ(n) = (p-1) * (q-1)$$
+4. Select PUBLIC KEY - $e$ , such that $e$ and $φ(n)$ are Co-primes, i.e, $$\gcd(e , φ(n))=1$$
+5. Calculate PRIVATE KEY, $d$ such that $$(d * e) \mod φ(n) = 1$$
+
+## Public and Private Keys
+
+1. The **Public  key** is `{ e , n }` which is known to all in the network.
```

Comment:
`{ ... }` is for sets, but this is a tuple?

Reply:
Done